components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/confirmation/ResendConfirmationManager.java

```diff
@@ -337,17 +357,21 @@ private void invalidateResendConfirmationCode(String resendCode, String notifica
      * @param user             User object
      * @throws IdentityRecoveryServerException Error storing recovery data
      */
-    private void addRecoveryDataObject(String secretKey, String recoveryData, RecoveryScenarios recoveryScenario,
-                                       RecoverySteps recoveryStep, User user)
+    private void addRecoveryDataObject(String secretKey, String recoveryFlowId, String recoveryData,
```

Comment:
update the method comment with new param

Reply:
Fixed with ab489ffd47d3bb04acd840bdff99979c579d20aa